yaml/pois.yaml

```diff
@@ -826,15 +858,29 @@ filters:
       tier: 6
   # hedge - no POI
   # pedestrian - no POI
-  # pitch
-  - filter: {leisure: pitch}
+  # pitch - either has a name, or shows at a higher zoom for some sports
+  - filter:
+      leisure: pitch
+      name: true
     # already clamped below area min
     min_zoom: 16
     output:
       <<: *output_properties
       kind: pitch
       kind_detail: {col: sport}
       tier: 6
+  # pitch - only show no-name pitches for some smaller set of sports, and at a
+  # higher zoom.
+  - filter:
+      leisure: pitch
+      sport: [baseball, basketball, football, hockey, soccer, tennis]
```

Comment:
The kind_detail were only listed in the example because that's what we have icons for... but I don't think that needs to be limited in the data like this. I'd remove this line and change the comment above.

Reply:
OK, removed the filter and adjusted tests in 5265c9d.